docker/test.ps1

```diff
@@ -0,0 +1,9 @@
+# This test script :
+# - prepares the database for rspec tests
+# - launches the rspec tests
+$DateTime=Get-Date -Format "yyyyMMdd-HHmmss"
+
+Write-Host "bundle exec rake db:test:prepare : prepare the database for rspec tests" -ForegroundColor Blue
+docker-compose.exe run web bundle exec rake db:test:prepare > log/test-prepare-$DateTime.log 2>&1
+Write-Host "bundle exec rspec spec : launch the rspec tests" -ForegroundColor Blue
+docker-compose.exe run web bundle exec rspec spec > log/test-rspec-$DateTime.log 2>&1
```

Comment:
This is slightly different from the way docker/test operates for linux, `docker/test` logs everything to `log/test-$DATE.log`

I think it's a good idea to split the logs into two files, could you update the `docker/test` /`docker/test-log` to do the same ?

Reply:
Done.
Although it is not part of the docker dev environment for windows.